### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by handplant** [verbb/formie#905]
**Description**
I have two required "Agree fields" in a simple form. When I submit the form once, everything validates as it should. On the second submission, the error `Uncaught RangeError: Maximum call stack size exceeded` comes up in Chrome Devtools.

**Steps to reproduce**
1. Create a new form
2. Create two required agree fields
3. Submit the form twice

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- PHP version: 8.0.9
- Plugin version: 1.5.17
- Craft version: Craft Pro 3.7.43
- Multi-site: No

The problem also exists with Craft Solo 4.0.3 and Plugin version: 2.0.0-beta.10

**Screenshots**
![Bildschirmfoto 2022-05-25 um 14 58 45](https://user-images.githubusercontent.com/2598962/170267698-6b1b8196-29f3-4cb8-8c0a-2118c34d6372.png)
![Bildschirmfoto 2022-05-25 um 14 59 07](https://user-images.githubusercontent.com/2598962/170267706-2dcc6a19-5746-42cf-8a91-fc087873bb61.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.18",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.5.19](https://github.com/verbb/formie/releases/tag/1.5.19)
